docs/features/populate.rst

```diff
@@ -0,0 +1,296 @@
+          name: MyOrg
+          label: MyOrg{{org_label_suffix}}
+
+
+To populate the system
+
+.. code-block:: console
+
+    (robottelo_env)[you@host robottelo]$ manage data populate test_data.yaml -v
+    2017-01-04 04:31:17 - robottelo.populate.base - INFO - CREATE: The first organization...
+    2017-01-04 04:31:19 - robottelo.populate.base - INFO - search: Organization {'query': {'search': 'name=MyOrg,label=MyOrg'}} found unique item
+    2017-01-04 04:31:19 - robottelo.populate.base - INFO - create: Entity already exists: Organization 36
+    2017-01-04 04:31:19 - robottelo.populate.base - INFO - registry: org_1 registered
+
+To validate the system
+
+.. code-block:: console
+
+    (robottelo_env)[you@host robottelo]$ manage data validate test_data.yaml
+    (robottelo_env)[you@host robottelo]$ echo $?
+    0  # system validated else 1
+
+
+Use :code:`$ manage data --help` and :code:`$ manage populate --help` for more info
+
+Decorator
+---------
+
+Other way to use populate is via decorator, it is useful to decorate a test_case
+forcing a populate or validate operation to be performed.
+
+    Having a data_file like::
+
+        actions:
+          - model: Organization
+            register: organization_1
+            data:
+              name: My Org
+
+    Then you can use in decorators::
+
+        @populate_with('file.yaml')
+        def test_case_(self):
+            'My Org exists in system test anything here'
+
+    And getting the populated entities inside the test_case::
+
+        @populate_with('file.yaml', context=True)
+        def test_case_(self, context):
+            assert context.entities.organization_1.name == 'My Org'
+
+    You can also set a name to the context argument::
+
+        @populate_with('file.yaml', context='my_context')
+        def test_case_(self, my_context):
+            assert my_context.organization_1.name == 'My Org'
+
+
+And if you dont want to have YAML file you can provide a dict::
+
+    data = {
+        'actions': [
+            {
+                'model': 'Organization',
+                'register': 'organization_1',
+                'data': {
+                    'name': 'My Organization 1',
+                    'label': 'my_organization_1'
+                }
+            },
+        ]
+    }
+
+
+    @populate_with(data, context=True, verbose=1)
+    def test_org_1(context):
+        """a test with populated data"""
+        assert context.registry['organization_1'].name == "MyOrganization1"
+
+
+The YAML data file
+------------------
+
+In the YAML data file it is possible to specify 3 sections, ``config``, ``vars`` and ``actions``.
+
+
+config
+++++++
+
+The config may be used to define special behavior of populator and its keys are:
+
+- populator
+
+  The name of the populator defined in ``populators``
+- populators
+
+  The specification of populator modules to be loaded
+- verbose
+
+  The verbosity of logging 0, 1 or 2, it can be overwritten with -vvv in commands.
```

Comment:
I'll fix the comment and example, thanks. currently there are 3 levels of verbosity, but I am not sure if it is right, open to changes.

0 - log nothing
1 - (-v) log only populate INFO
2 - (-vv) log populate DEBUG and nailgun DEBUG
3 - (-vvv) log everything including ssh (which will be added when CLI support impkemented)